images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
Yeah, I mentioned that to @simaishi and she mentioned the SCL one. Not sure if that was just a testing effort or some other issue?